### PR TITLE
Make articles go faster

### PR DESCRIPTION
**Pull request by matthew-andrews**

### CODE REVIEW

File: client/components/comments/main.js [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
 var oComments = require('o-comments');
+var fetchres = require('fetchres');
 
-module.exports = {
-
-	init: function (uuid, flags) {
-
-		if (flags.get('articleComments').isSwitchedOn && document.getElementById('comments')) {
+module.exports = {};
+module.exports.init = function (uuid, flags) {
```
Comment (matthew-andrews) [inline client/components/comments/main.js]: with this i just wanted the indentation to be no more than it was (with all the ifs and thens it got quite deep — not objecting to the way you did it originally @ironsidevsquincy )

File: server/app.js [hunk 2]
```diff
@@ -15,8 +15,9 @@ app.use(/^\/([a-f0-9]+\-[a-f0-9]+\-[a-f0-9]+\-[a-f0-9]+\-[a-f0-9]+)$/, require('
 app.get(/^\/fastft\/([0-9]+)(\/[\w\-])?/, require('./controllers/fastft'));
 app.get(/^\/([a-f0-9]+\-[a-f0-9]+\-[a-f0-9]+\-[a-f0-9]+\-[a-f0-9]+)$/, require('./controllers/capi'));
 
-app.get('/:id(' + articleUuidRegex + ')/people', require('./controllers/related/people'));
-app.get('/:id(' + articleUuidRegex + ')/organisations', require('./controllers/related/organisations'));
+app.get('^/:id(' + articleUuidRegex + ')/people', require('./controllers/related/people'));
+app.get('^/:id(' + articleUuidRegex + ')/organisations', require('./controllers/related/organisations'));
+app.get('^/:id(' + articleUuidRegex + ')/comments-hack', require('./controllers/comments-hack'));
```
Comment (matthew-andrews) [inline server/app.js]: this is a nice pattern — we should do this approach for all uuid-y endpoints

File: client/components/comments/main.js [hunk 3]
```diff
@@ -1,12 +1,19 @@
...
 var oComments = require('o-comments');
+var fetchres = require('fetchres');
 
-module.exports = {
-
-	init: function (uuid, flags) {
-
-		if (flags.get('articleComments').isSwitchedOn && document.getElementById('comments')) {
+module.exports = {};
+module.exports.init = function(uuid, flags) {
```
Comment (ironsidevsquincy) [inline client/components/comments/main.js]: :+1:

**Comment by ironsidevsquincy** [comment]
:+1:

**Pull request merged**